src/explorer/App.test.js

```diff
@@ -111,6 +111,14 @@ describe("explorer/App", () => {
     expect(link.prop("href")).toMatch(/https?:\/\//);
   });
 
+  it("should have a project title", () => {
```

Comment:
Assuming that we stick with a ProjectDetail component, there should be two tests.

1. The containing component instantiates a ProjectDetail with the right props
2. The ProjectDetail renders an example RepoId correctly

Reply:
Yep, exactly. It's nice to have very narrowly scoped tests, makes it easy to understand and resolve failures.